api/server/server.go

```diff
@@ -174,7 +170,7 @@ func NewServer(conf Config) (*Server, error) {
 	lchain := lotuschain.New(c)
 	var ms ffs.MinerSelector
 	if conf.Embedded {
-		ms = fixed.New([]fixed.Miner{{Addr: "t0300", EpochPrice: 4000000}})
+		ms = fixed.New([]fixed.Miner{{Addr: "t01000", EpochPrice: 1000000}})
```

Comment:
Necessary consequences of switching to `testnet/3`. Similar changes are in the wild.